From e6cdcf0b1e0bb7c1e973c2df8d3d068da53916b3 Mon Sep 17 00:00:00 2001
From: Maksym Sditanov <maxim.sditanov@lge.com>
Date: Thu, 9 Jan 2014 16:28:58 +0200
Subject: [PATCH 1/1] Add to pkgconfig file pbnjson library dependency

:Release Notes:
Add linking to pbnjson library

:Detailed Notes:
PmLogLib library have external dependency from pbnjson. So, any
component (like luna-service2, db8) should obviously add linking
to pbnjson. To avoid potential linking problems it is usefull
to specify into PmLogLib linking flags external dependency from
pbnjson

:Testing Performed:
Compiled locally and tested localy

:QA Notes:

:Issues Addressed:

Open-webOS-DCO-1.0-Signed-off-by: Maksym Sditanov <maxim.sditanov@lge.com>

Change-Id: Ida411518d878fb9727a65db63ee82f9ebd8967d5
---
 files/pkgconfig/PmLogLib.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/files/pkgconfig/PmLogLib.pc.in b/files/pkgconfig/PmLogLib.pc.in
index 637c77f..f5b515a 100644
--- a/files/pkgconfig/PmLogLib.pc.in
+++ b/files/pkgconfig/PmLogLib.pc.in
@@ -24,3 +24,4 @@ Description: @WEBOS_PROJECT_SUMMARY@
 Version: @WEBOS_COMPONENT_VERSION@
 Libs: -L${libdir} -l@CMAKE_PROJECT_NAME@
 Cflags: -I${includedir}
+Requires: pbnjson_c
-- 
1.8.3.2

